src/hooks/useAvailableMembers.ts

```diff
@@ -31,25 +29,19 @@ export const useAvailableMembers = (
   credential: AuthorizationCredential,
   resourceId?: string,
   parentCommunityId?: string,
-  parentMembers?: Member[], // Just because the organizations doesn't have community.
-  limitFirstUserFetch = 10
+  parentMembers?: Member[] // Just because the organizations doesn't have community.
 ): AvailableMembersResults => {
   const { hubId, loading: loadingHub } = useHub();
   const {
-    data: _allUsers,
+    data: usersQueryData,
     loading: loadingUsers,
     error: userError,
-    fetchMore,
-  } = useUsersDisplayNameQuery({
-    variables: { first: limitFirstUserFetch },
+  } = useUsersQuery({
```

Comment:
Please remove this query and its usage (1 usage) from the app, because it's very heavy.
You just need `id` and `displayName` here.